redis/src/aio.rs

```diff
@@ -1079,7 +1083,25 @@ mod connection_manager {
         ///
         /// This requires the `connection-manager` feature, which will also pull in
         /// the Tokio executor.
+        #[deprecated = "use new_with_backoff"]
```

Comment:
Do we need to deprecate here? Can we just supply a reasonable default and keep the method?

Reply:
We can do that, but then we need to either give a good default, which means that we're changing the behavior of the class without informing the users, or we give a default that matches the current behavior (that's what I did), which is a bad default. My preference is to make sure that users are aware of the new function without changing the behavior under their feet. If you prefer to give a good default, that's also fine.

Reply:
I think it's fine to provide a "good" default -- I think retries are the sort of thing we'd expect a `ConnectionManager` to do, so I think we should do that rather than deprecate. Everyone reads the Changelog, right? :grinning:

Reply:
done